"make tools-all" should allow building tools such as mkimage and the new
imximage without any config, but imximage.c currently fails to build
with:
imximage.h:27:20: error: config.h: No such file or directory

config.h is not needed in imximage.h nor in imximage.c, and imximage.h
is only included from imximage.c, so drop this include to fix the build.

Signed-off-by: Lo?c Minier <loic.minier@linaro.org>
---
 tools/imximage.h |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/tools/imximage.h b/tools/imximage.h
index 38ca6be..d126a46 100644
--- a/tools/imximage.h
+++ b/tools/imximage.h
@@ -24,8 +24,6 @@
 #ifndef _IMXIMAGE_H_
 #define _IMXIMAGE_H_
 
-#include <config.h>
-
 #define MAX_HW_CFG_SIZE_V2 121 /* Max number of registers imx can set for v2 */
 #define MAX_HW_CFG_SIZE_V1 60  /* Max number of registers imx can set for v1 */
 #define APP_CODE_BARKER	0xB1
-- 1.7.2.3
